data/governments.txt

```diff
@@ -516,8 +530,25 @@ government "Hai (Friendly Unfettered)"
 		"Pirate (Devil-Run Gang)" -.01
 		"Korath" -.01
 		"Hai" .01
+		"Hai (Wormhole Access)" .01
+		"Hai (Unfettered)" 1
 		"Bounty Hunter that Attacks Hai" -.01
-		"Elenctic Commune" .1
+		"Elenctic Commune" .01
+	"bribe" .02
+	"fine" 0
+	"friendly hail" "friendly unfettered"
+	"friendly disabled hail" "friendly disabled unfettered"
+	"hostile hail" "hostile unfettered"
+	"hostile disabled hail" "hostile disabled unfettered"
+
+government "Hai (Unfettered Civilians)"
+	swizzle 4
+	color "governments: Hai (Unfettered)"
+	"display name" "Hai (Unfettered Logistics)"
```

Comment:
```suggestion
	"display name" "Hai (Unfettered)"
```
I'm not sure about this but I think we should change it here and mb worry about it in another PR
All Hai governments shouldnt have different names I think btw, but this isnt the PR for that